src/palace/manager/search/external_search.py

```diff
@@ -2104,7 +2105,20 @@ def featurability_scoring_functions(self, random_seed):
         nested = Nested(path="licensepools", query=available)
         available_now = dict(filter=nested, weight=5)
 
-        function_scores = [quality_field, available_now]
+        # Works with higher lane priority scores are more featurable
+        lane_priority_level = SF(
+            "field_value_factor",
+            field="licensepools.lane_priority_level",
+            factor=10,
+            modifier="none",
+            missing=IntegrationConfigurationConstants.DEFAULT_LANE_PRIORITY_LEVEL,  # assume default if missing
+        )
+
+        function_scores = [
+            quality_field,
+            available_now,
+            lane_priority_level,
```

Comment:
If I remove the lane_priority_level function from the function_scores list, I get the same results.

Reply:
Even removing the quality_field and available_now functions, I still get the same results.

Reply:
Based on our discussion the other day, I'm assuming that this conversation is resolved.